dipy/viz/tests/test_ui.py

```diff
@@ -539,13 +539,38 @@ def test_ui_range_slider(interactive=False):
 
 @npt.dec.skipif(not have_vtk or skip_it)
 @xvfb_it
-def test_ui_listbox_2d(recording=False):
+def test_ui_listbox_2d(interactive=False):
```

Comment:
Like we discussed in PR #1559 , we may want to have a `mode` parameter that can either accep `None, `interactive` or `recording`.